src/commands/getCommand.ts

```diff
@@ -0,0 +1,27 @@
+import Base from './base'
+import { Flags } from '@oclif/core'
+
+export default abstract class GetCommand extends Base {
+    authRequired = true
+
+    static flags = {
+        ...Base.flags,
+        'sortBy': Flags.string({
+            description: 'Sort By',
+            options: [
+                '_id',
```

Comment:
I'm not sure sorting by mongo id would be very useful. We can probably get rid of this as an option

Reply:
yes, I'm not sure about that too, but since our api accepts this option, do you think we should keep it here too?

Reply:
technically our api accepts all the fields as options for sorting. But i  think we should remove this as an option. It's an internal db field for the project so i don't think sorting by this would useful